remoteworkitem/remoteworkitem.adoc

```diff
@@ -0,0 +1,86 @@
+= Remote Work Items
+
+:toc:
+:sectnums:
+:experimental:
+
+== Overview
+
+The remote work item is a work item that is tracked remotely. The
+remote work item implemented using three major components:
+
+Tracker:: The basic configuration for a remote tracker (type, URL)
+Tracker Query:: The tracker specific query and schedule to fetch and import remote items
+Tracker Item:: Temporary staging storage from where remote work item gets created
+
+== Tracker
+
+To create a new tracker configuration:
+
+[source, bash]
+./bin/alm-cli create tracker --payload '{"type": "github", "url": "https://api.github.com/"}' -H localhost:8080
+
+To update an existing tracker configuration:
+
+[source, bash]
+./bin/alm-cli update tracker --id=1 --payload '{"type": "github", "url": "https://api.local.example.com/"}' -H localhost:8080
+
+To list all tracker configurations:
+
+[source, bash]
+./bin/alm-cli list tracker -H localhost:8080
+
+To show one tracker configuration:
+
+[source, bash]
+./bin/alm-cli show tracker --id=1 -H localhost:8080
```

Comment:
In the preview mode on GH source examples are not rendered properly. Shouldn't you wrap them as follows to be on the safe side?

```
---
[source, bash]
./bin/alm-cli show tracker --id=1 -H localhost:8080
---
```